packages/ui/src/components/form-input/components/form-multi-select-v2.tsx

```diff
@@ -0,0 +1,55 @@
+import { forwardRef } from 'react'
+import { Controller, useFormContext } from 'react-hook-form'
+
+import { MultiSelect, type MultiSelectOption, type MultiSelectRef } from '@/components/multi-select-v2'
+
+interface FormMultiSelectV2PropsType
```

Comment:
i think the form wrapper should also accept other form-related props like label, placeholder, error, warning, caption etc